### PR TITLE
French: Use cases

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Updates French (`fr`) **use cases** with latest from Crowdin

### CODE REVIEW

File: src/content/translations/fr/governance/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+La gouvernance d'Ethereum n'est pas rigoureusement définie. Divers participants de la communauté ont des points de vue différents à ce sujet. Voici quelques-unes d'entre elles :
+
+- [Notes on Blockchain Governance](https://vitalik.ca/general/2017/12/17/voting.html) - _Vitalik Buterin_
+- [Governance on Ethereum](https://docs.ethhub.io/ethereum-basics/governance/) – _ETHHub_
+- [How does Ethereum Governance work?](https://cryptotesters.com/blog/ethereum-governance) – _Cryptotesters_
+- [How Ethereum governance works](https://medium.com/coinmonks/how-ethereum-governance-works-71856426b63a) – _Micah Zoltu_
+- [What is an Ethereum core developer?](https://hudsonjameson.com/2020-06-22-what-is-an-ethereum-core-developer/) - _Hudson Jameson_
+- [Governance, Part 2: Plutocracy Is Still Bad](https://vitalik.ca/general/2018/03/28/plutocracy.html) - _Vitalik Buterin_
+- [Moving beyond coin voting governance](https://vitalik.ca/general/2021/08/16/voting3.html) - _Vitalik Buterin_
```
Comment (minimalsm) [inline src/content/translations/fr/governance/index.md:184]: I think we decided that we **would** translate external article titles? @lukassim 

Screenshot 2022-02-24 at 14 55 24


https://ethereumorgwebsitedev01-frusecases.gtsb.io/fr/governance

File: src/content/translations/fr/security/index.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+- [Protecting Yourself and Your Funds](https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds) - _MyCrypto_
+- [4 Ways to Stay Safe in Crypto](https://www.coindesk.com/tech/2021/04/20/4-ways-to-stay-safe-in-crypto/) - _CoinDesk_
+- [Security Guide For Dummies And Smart People Too](https://medium.com/mycrypto/mycryptos-security-guide-for-dummies-and-smart-people-too-ab178299c82e) - _MyCrypto_
+- [Crypto Security: Passwords and Authentication](https://www.youtube.com/watch?v=m8jlnZuV1i4) - _Andreas M. Antonopoulos_
+
+### Prévention des arnaques {#reading-scam-education}
+
+- [Staying Safe: Common Scams](https://support.mycrypto.com/staying-safe/common-scams) - _MyCrypto_
+- [Avoiding Scams](https://bitcoin.org/en/scams) _Bitcoin.org_
```
Comment (minimalsm) [inline src/content/translations/fr/security/index.md:280]: Same here

File: src/content/translations/fr/nft/index.md [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+
+## Complément d'information {#further-reading}
+
+- [Crypto art data](https://cryptoart.io/data) – Richard Chen, updated automatically
+- [OpenSea: the NFT Bible](https://opensea.io/blog/guides/non-fungible-tokens/) – _Devin Fizner, 10 Janvier 2020_
+- [A beginner's guide to NFTs](https://linda.mirror.xyz/df649d61efb92c910464a4e74ae213c4cab150b9cbcc4b7fb6090fc77881a95d) – _Linda Xie, Janvier 2020_
+- [Everything you need to know about the metaverse](https://foundation.app/blog/enter-the-metaverse) – _Foundation team, foundation.app_
+- [No, CryptoArtists Aren’t Harming the Planet](https://medium.com/superrare/no-cryptoartists-arent-harming-the-planet-43182f72fc61)
+- [A country's worth of power, no more](https://blog.ethereum.org/2021/05/18/country-power-no-more/) – _Carl Beekhuizen, 18 Mai 2021_
```
Comment (minimalsm) [inline src/content/translations/fr/nft/index.md:369]: These should be translated

**Review by minimalsm — COMMENTED** [review]
These look fine other than the external English links not being translated. @lukassim should we just merge this and we can deal with Acolad to resolve later?

**Comment by lukassim** [comment]
> These look fine other than the external English links not being translated. @lukassim should we just merge this and we can deal with Acolad to resolve later?

I think we can merge for now, and I will let Acolad know that these should be translated during the next batch. I will also mention that the linguists should pay attention to this and translate article titles, even if our translators leave some in Englih.

**Review by corwintines — APPROVED** [review]
Approved based on comments

**Pull request merged**